src/test/shell/integration/action_env_test.sh

```diff
@@ -124,6 +124,31 @@ function test_simple_latest_wins() {
   expect_log "BAR=bar"
 }
 
+function test_simple_latest_wins_removed() {
```

Comment:
Can you also add tests to verify that `--action_env=FOO` and `--action_env=FOO=` work correctly?

(Same for `--repo_env`; `--run_env` is already covered)

Reply:
The `--action_env=FOO` case was already covered by `test_client_env`, but I added a test for `--action_env=FOO=` as well as `--repo_env`.